pages/public_cloud/public_cloud_network_services/technical-resources-03-lb_logs_2_customers/guide.en-gb.md

```diff
@@ -0,0 +1,201 @@
+- an up-and-running Public Cloud Load Balancer. This [guide](../getting-started-01-create-lb-service/) describes how to configure one.
+- Both LDP account and Public Cloud Load Balancer must belong to the same OVHcloud account.
+
+## Concepts & limits
+
+> [!warning]
+> To date the logs of the **UDP** listeners are not forwarded 
+
+What are the logs of a Public Cloud Load Balancer ?
+
+The forwarded logs are generated by [HAproxy](https://en.wikipedia.org/wiki/HAProxy) (the opensource component that is used to make the load balancing).
+
+The log messages contains the following fields:
+| Field name | Description | Unit |
+|------------|-------------|---------| 
+| project_id | the id of the public cloud project to which the load balancer belongs | uuid |
+| load_balancer_id | the id of the load balancer that received the request/connection | uuid |
+| listener_id | the id of the listener that received the request/connection | uuid |
+| client_ip | IP address of the client which initiated the TCP connection to the Load Balancer | IP |
+| client_port | TCP port of the client which initiated the TCP connection to the Load Balancer | Numeric |
+| accept_date | The timestamp at which the request / connection was made | datetime (with millisecond resolution) e.g. 25/Mar/2024:14:07:19.536|
+| http_verb http_request http_version | The HTTP request | String e.g "GET / HTTP/1.1"|
+| http_status | The returned HTTP status | Numeric |
+| bytes_read | The number of bytes read by the server | Numeric |
+| bytes_uploaded | The number of bytes sent by the server to the client | Numeric |
+| client_certificate_verify | Returns the verify result error ID when the incoming connection was made over an SSL/TLS transport layer, otherwise zero if no error is encountered. | Numeric |
+| client_certificate_distinguished_name | When the incoming connection was made over an SSL/TLS transport layer, returns the full distinguished name of the subject of the certificate presented by the client | String |
+| pool_id:listener_id | the pool to which the request/connection was redirected | uuid| 
+| member_id | the member to which the request / connection was sent | uuid |
+| total_session_duration_time | the time in milliseconds the TCP session has been opened when this request is made | Numeric |
+| termination_state | session termination indicator: 2 letters for TCP, 4 letters for HTTP all the details on [HAProxy documentation page](https://docs.haproxy.org/2.6/configuration.html#8.5)|String (e.g. "----")|
+
+
+The following fields are computed from `client_ip` and provided in logs:
+| Field name | Description | Unit |
+|------------|-------------|---------| 
+|client_ip_city_name | the city computed by Logstash Geoip from `client_ip`| String (e.g. `Lille`) |
```

Comment:
These fields are computed by "Graylog" directly not "Logstash". Otherwise these additional fields would show up in [Logs Data Platform Websocket live tail](https://help.ovhcloud.com/csm/fr-logs-data-platform-ldp-tail?id=kb_article_view&sysparm_article=KB0037675) but they don't because they are added by Graylog.